docs/concepts/libp2p.md

```diff
@@ -33,43 +33,43 @@ Libp2p works with a lot of different addressing schemes in a consistent way. A m
 
 The technology used to move your data from one machine to another. Transports are defined in terms of two core operations, _listening_ and _dialing_. Listening means that you can accept incoming connections from other peers. _Dialing_ is the process of opening an outgoing connection to a listening peer. One of Libp2p's core requirements is to be _transport agnostic_, meaning the decision of what transport protocol to use is up to an application's developer (who may decide to support many different _transports_ at the same time).
 
-#### [Security](https://docs.libp2p.io/introduction/what-is-libp2p/#security)
+#### [Security](https://docs.libp2p.io/introduction/#secure-communication)
 
 Libp2p supports upgrading a transport connection into a securely encrypted channel. You can then trust the identity of the peer you're communicating with and that no third-party can read the conversation or alter it in-flight. The current default is [TLS 1.3](https://www.ietf.org/blog/tls13/) as of IPFS 0.7. The previous default of [SECIO](https://docs.libp2p.io/concepts/secure-comms/) is now deprecated and disabled by default (see [this blog post](https://blog.ipfs.tech/2020-08-07-deprecating-secio/) for more information).
 
-#### [Peer identity](https://docs.libp2p.io/concepts/peer-id/)
+#### [Peer identity](https://docs.libp2p.io/reference/glossary/#peerid)
```

Comment:
For this, we recently changed the page to a general doc on "peers", forgot to apply the redirect but it is being applied in: https://github.com/libp2p/docs/pull/186. However, the correct link would be: https://docs.libp2p.io/concepts/peers/#peer-id